### DISCUSSION TITLE
Compteur incompatible Il doit collecter les données listées ci dessus

### DISCUSSION THREAD

**Discussion by antoinetielbeke** [bokub/conso-api#19] [Aide] [answered]
I do not speak French so please excuse me for the English :)

I am trying to setup Conso API for my Linky energy reader but when I want to give Conso access to my Enedis it says ` Compteur incompatible Il doit collecter les données listées ci dessus`. If I understand correctly, it says my energy reader is not compatible for "Electricity delivery point (PDL or PRM)". But when I check "Le Compteur" it says:

- Type de compteur: Compteur Linky communicant
- Type de comptage: Consommation électricité 

So why is it not compatible for conso api? Can I do/request anything for this to become available?

Thanks in advance!

**Comment by bokub** [comment, marked as answer]
Hi,

It's a duplicate of #17 and #18 , the Enedis page is just broken. It's very frequent, the only thing to is is to be patient until they fix their mess !

**Reply by antoinetielbeke** [reply]
Thanks for the quick reply and sorry for the duplicate issue. Normally I would check but language barrier and stuff 😅
